bitnami/kube-prometheus/templates/exporters/core-dns/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if .Values.coreDns.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "kube-prometheus.fullname" . }}-coredns
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: {{ template "kube-prometheus.fullname" . }}-coredns
+spec:
+  jobLabel: k8s-app
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: {{ template "kube-prometheus.fullname" . }}-coredns
+  namespaceSelector:
+    matchNames:
+      - "kube-system"
```

Comment:
```suggestion
      - {{ .Values.coreDns.namespace }}
```

Reply:
Good point!
I prepared the values but forgot to make change here...
Corrected already.